lib/solidus_dev_support/solidus_command.rb

```diff
@@ -13,6 +13,12 @@ class SolidusCommand < Thor
     desc 'e', 'Manage solidus extensions (shortcut for "extension")'
     subcommand 'e', Extension
 
+    desc 'version', 'Displays solidus_dev_support version'
+    def version
+      puts "Solidus Dev Support version #{SolidusDevSupport::VERSION}"
```

Comment:
Since the command would be `solidus version` I would expect this to also show the (current) solidus version, and maybe also the solidus_support version (which is different from solidus and is still an important piece of the machinery).

Reply:
@elia I added the solidus version. I wasn't sure if the `solidus_support` would always be available. So, I decided to keep it out for now.